neurom/core/morphology.py

```diff
@@ -386,7 +400,7 @@ def volume(self):
         """
         return sum(s.volume for s in self.iter_sections())
 
-    def iter_sections(self, order=Section.ipreorder, neurite_order=NeuriteIter.FileOrder):
```

Comment:
Please don't break the public api :)

Reply:
But this API makes no sense :laughing: 
`self` is a `Neurite` object so the `neurite_order` parameter is just never considered.
If this change breaks something it means that the user tries to do something very weird and the result is probably not what he has in mind. But this change can go to the v4 branch if you prefer.

Reply:
Indeed, I agree it makes no sense. Let's add it on v4 yeah.

Reply:
Ok, I am going to open a new PR on the v4 branch

Reply:
Could you keep the iterato_type instead of order for consistency with the function, please?

Reply:
Ah no I can just change the target here

Reply:
Btw, why is it called `iterator_type`? Wouldn't `section_order` be more relevant?

Reply:
I like both, but the reason I suggested the change is to be as consistent as possible with the existing terminology.

Reply:
I agree it must be consistent, I just think that since it's a new major version we could take the opportunity to improve the API. And in this case, `iterator_type` looks a bit obscure to me while `section_order` looks more clear for the user.